### DISCUSSION TITLE
Does this support Forge?

### DISCUSSION THREAD

**Discussion by tap90** [cswendrowski/FoundryVTT-vox-ludos-speech#7] [General]
Hi is currently this module supported?

Does it work under the Forge platform? cause I have made an account on Azure Portal and activated speech service as requested by this module. Copy the Subscription key in settings and set the correct location but I receive two errors during the use of this module.

If I press f12 I can see in the console these errors:

DevTools failed to load source map: Could not load content for https://assets.forge-vtt.com/bazaar/modules/voxludos/1.1.0/scripts/lib/microsoft.cognitiveservices.speech.sdk.bundle.js.map: HTTP error: status code 404, net::ERR_HTTP_RESPONSE_CODE_FAILURE

And another one:

WebSocket connection to 'wss://eastus.tts.speech.microsoft.com/cognitiveservices/websocket/v1?Ocp-Apim-Subscription-Key=58f73e54276948759b739b6138e81757&X-ConnectionId=43043B8C61B14FED9CAA90A7E90ECF8D' failed:

**Comment by hmqgg** [comment]
Source map is not necessary for this module to work.
It still works for me, and it *should* work on Forge.
May you try another API key or regenerate one with Azure Speech service?

**Comment by tap90** [comment]
Thanks for reply

I have regenerated the key, the error still exist, and it doesn't work!

**Reply by hmqgg** [reply]
Ensure it is a `Speech service` and its location/region must match.

**Comment by tap90** [comment]
It was correct but now seems to work and I don't have to change anything, maybe the account was not valid yet
